doc/modules/neural_networks.rst

```diff
@@ -1,11 +1,140 @@
 .. _neural_network:
 
+==================================
+Neural network models (supervised)
+==================================
+
+.. currentmodule:: sklearn.neural_network
+
+Randomly weighted neural networks
```

Comment:
Should I remove  all documentation on the randomly weighted neural network algorithm ? The random basis function section below only talks about the mapping function but not about how it can be used for supervised learning algorithms. What do you suggest ?